src/sshfs_mount/sshfs_mount.cpp

```diff
@@ -26,37 +26,93 @@
 #include <multipass/format.h>
 
 #include <iostream>
+#include <sstream>
 
 namespace mp = multipass;
 namespace mpl = multipass::logging;
 
 namespace
 {
 constexpr auto category = "sshfs mount";
-template <typename Callable>
-auto run_cmd(mp::SSHSession& session, std::string&& cmd, Callable&& error_handler)
+const std::string fuse_version_string{"FUSE library version "};
```

Comment:
In a current multipass instance:
```
$ sshfs -V
...
FUSE library version: 2.9.7
$
```
The colon expected?

Reply:
D'oh, silly fragile output strings...

Reply:
Ugh, come to find out, the archive version of `sshfs -V` prints out some of the info on `stdout` as well as `stderr` for whatever reason and we are only reading `stdout` when getting output from a command running through `libssh`.  We are going to have to modify how we are getting output through `libssh` and read both `stdout` and `stderr` for any given command regardless if there is an error or not returned by the command.

Reply:
And of course `FUSE library version:` is one of those that is written to `stderr` :frowning_face:

Reply:
Ok, fix seems easy- just concatenate reading `stdout` and `stderr` in `run_cmd()`.

Reply:
Ok, I fixed this by not including the space in `fuse_version_string` along with fixing up `run_cmd()` like mentioned previously.  Lastly, I use a regex to split on the returned string.